projectDocs/translating/crowdin.md

```diff
@@ -160,11 +145,53 @@ In Crowdin, this information appears at the end of the context section.
 1. From the language list, select your language (if it is listed), press `enter` and say yes when you're asked to restart NVDA.
 1. The messages you have translated should now be heard or brailled in your native language provided that the synthesizer you are using supports your language or a braille code for your language exists.
 
 Whenever you add or update your interface translations, repeat the steps above (copying the updated .mo file and restarting NVDA) to test your updated translation messages.
+
+## Translating NvDA' s user documentation
+
+Documentation available for translation includes:
+* The NVDA user guide (userGuide.xliff)
+* The NVDA What's New document (changes.xliff)
+
+To translate any of these files:
+
+* Download the xliff file from the Files section of your language on Crowdin.
+    * Make sure to choose "Download" not "Export xliff".
+* Make a copy of this file.
+* Open the po file in Poedit, translate, and save the file.
+* Use the nvdaL10nUtil program to strip the xliff so that it only contains translations that were added / changed. E.g.
+```
+nvdaL10nUtil stripXliff -o <old xliff file> <translated xliff file> <output stripped xliff file>
+```
+* Upload the xliff file back to Crowdin. If it is a stripped xliff file, it is safe to check the `allow target to match source` checkbox.
+
+Alternatively, you can use the [Crowdin interface directly](https://support.crowdin.com/online-editor/).
+
+### Translating markdown
+The English NVDA user documentation is written in markdown syntax.
+The xliff file you are directly translating has been generated from that markdown file.
+It contains the content of any line that requires translation, shown in the order it appears in the original markdown file.
+
+Structural lines that do not contain any translatable content (such as blank lines, hidden table header rows, table header body separator lines etc) are not included here.
+
+Structural syntax from the beginning and end of lines (such as heading prefix like `###`, heading anchors like `{#Introduction}`, and initial and final vertical bars on table rows) has been removed from the content to translate, but is available to view in the translator notes for that line.
+Content may still however contain inline markdown syntax such as links, inline code fences (`\``), and table column separators (`|`).
+This syntax must be kept in tact when translating.
+
+All strings for translation contain translator notes which include:
+* Line: the original line number in the markdown file.
+* prefix: any structural markdown on the line before this content.
+* Suffix: any structural markdown on the line after this content.
+
+### Verifying your translation
+When ever you have saved the xliff file with Poedit, you can use the nvdaL10nUtil program to generate the html version of the documentation file. E.g.
+```
+nvdaL10nUtil xliff2html -t [userGuide|changes|keyCommands] -l <lang> <xliff file> <output html file>
```

Comment:
Why is the `-l <lang>` parameter requested here, whereas the language is already stored in the xliff file?

Reply:
good point. For xliff2html at least, this can be fetched. Obviously it will still be required for md2html, though that command won't probably be used directly by translators that much.